scripts/kind-e2e/e2e.sh

```diff
@@ -37,7 +37,7 @@ with_context cluster3 subctl_install_subm
 
 verify="--connectivity"
 if [[ "$lighthouse" == "true" ]]; then
-    verify="--all"
+    verify="--connectivity=false --service-discovery"
```

Comment:
So if deployed with LH it'll only check the LH things?

Reply:
Yeah, the idea is that the other combination already checked the connectivity stuff, why repeat?

Reply:
Why --connectivity=false now? why's that flag necessary?

Reply:
Oh, I see that's "true" by default

Can we change to use the first argument as the type of connectivity?, or make them sub-commands?

Reply:
> Yeah, the idea is that the other combination already checked the connectivity stuff, why repeat?

Sure in CI, but locally of course it'll not work like that and you'll only get LH without basic connectivity tests. If that's good or bad IDK, but it'll happen

Reply:
@mkolesnik which is fine coz LH should test only discovery, not connectivity. If someone wants to test LH with basic connectivity locally they can run with `-all`.

Reply:
> Oh, I see that's "true" by default
> 
> Can we change to use the first argument as the type of connectivity?, or make them sub-commands?

subcommands sounds like good idea, will add it in next patch.

Reply:
> @mkolesnik which is fine coz LH should test only discovery, not connectivity. If someone wants to test LH with basic connectivity locally they can run with `-all`.

How would you run it? `make e2e <what goes here?>`

Reply:
@mkolesnik Not make, these options are added to `verify` Locally one would normally do `make deploy` and then run `verify`

Reply:
OK but this is part of `make e2e` call and you can run that one locally not only in CI..

Reply:
Done. `connectivity` is true by default, so we just set `service-discovery`